### ISSUE TITLE
Meet requirements for submitting to carpentries lab 

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#264]
(https://github.com/carpentries-lab/reviews/blob/main/docs/reviewer_guide.md#reviewer-checklist)

**Comment by svenvanderburg** [comment]
I've created:
* #271 


One of the requirements is challenging for us:
_The lesson does not make use of superfluous data sets, e.g. increasing cognitive load for learners by introducing a new data set instead of reusing another that is already present in the lesson._
We have a few datasets, but I think in our case it brings added value to the course. You learn how to apply deep learning to different types of data and we need that type of data for the things we want to show in the specific episodes.

**Comment by svenvanderburg** [comment]
@dsmits can you check if you agree on that we already tick the boxes that I ticked above?

**Comment by svenvanderburg** [comment]
@tobyhodges if you have time you can also have a look if you agree?

**Comment by dsmits** [comment]
Discussed with @svenvanderburg, we haven't checked whether the objectives defined in every episode have been covered. It's quite some work to check that so let's turn that into a separate issue

**Comment by dsmits** [comment]
I think we still need to check the diagnostic power of all exercises

**Comment by dsmits** [comment]
Also this point:
 "Learning objectives for the lesson and its episodes are clear, descriptive, and measurable. They focus on the skills being taught and not the functions/tools e.g. “filter the rows of a data frame based on the contents of one or more columns,” rather than “use the filter function on a data frame."
 
 We need to check that for the objectives

**Comment by dsmits** [comment]
Lastly, regarding the setup instructions, I have some beef with it.
I think we should adapt them so that people create a new conda environment. I think that would be necessary to conform to " The setup and installation instructions are complete, accurate, and easy to follow."

**Comment by tobyhodges** [comment]
I agree with all of @dsmits' comments above. A few thoughts to add from my perspective:

1. Coverage of objectives and diagnostic power of exercises: in my view, the lesson has quite a lot of objectives and relatively few exercises. The authors have preferred to take a "guided tour" approach, of demonstrating techniques and having learners follow along. That works fine, and offers opportunities for formative assessment, if delivered in a workshop in a very interactive way. However, it relies on that interactivity for learners to actually be _applying_ what they are learning, rather than only copying what the instructor is showing on their screen. The lesson would probably benefit from more exercises, to provide instructors with more opportunities to get measurable feedback on how well learners have grasped what was intended with each objective for a given episode.
    - one example, based on the objective "Plot the loss curve of the training process" in https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html: the episode includes an exercise ([_The Training Curve_](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#the-training-curve)) that is more suited to an objective like "evaluate the configuration of a training process with a loss curve" (another note - and I will open an issue for this - that the solution provided for this exercise could be improved, e.g. by adding an example of a loss plot for a poorly-configured training process, and by explaining what "jitter" in the curve might indicate/how that could be addressed, etc.)
2. In my view, the choice of different datasets in the lesson can be justified. It is often the case that for these kinds of "introduction to a broadly-applicable technique" lessons, you need to include multiple datasets to illustrate the generally important concepts and to guard against a kind of "overfitting" of the lesson itself to a particular example. But of course the lesson reviewers will have their own perspective on this when the lesson is in review!
3.

**Comment by svenvanderburg** [comment]
Thanks @dsmits and @tobyhodges for your great and useful comments! 🙏 I created new issues based on your comments and updated my comment above (https://github.com/carpentries-incubator/deep-learning-intro/issues/264#issuecomment-1332089064).

**Comment by svenvanderburg** [comment]
This only awaits https://github.com/carpentries-incubator/deep-learning-intro/issues/323, which is done if #322 is merged

**Comment by svenvanderburg** [comment]
Done!
